Add large hunspell dicts

Provessor opened the pull request:
<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#sec-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
###### Motivation for this change
Larger dictionary options for American, Canadian and Australian English

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

Provessor committed:
Move en_AU alias to above declaration to match others

mmahut commented:
Can you please squash your commits and include the package name in the commit message, as asked in https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md ?

Provessor committed:
Add en_GB-large, en_US-large, en_CA-large and en_AU-large hunspell dictionaries

Squashed commit of the following:

commit e3c0054263f6d71c1890ecb52b06ade71ce046ee
Author: Jai Flack <jaiflack@protonmail.com>
Date:   Tue Aug 27 18:38:34 2019 +1000

    Add en_GB-large, en_US-large, en_CA-large and en_AU-large hunspell
    dictionaries

    Squashed commit of the following:

    commit 6b16e7326061aa55bb6993b3913fb4fb701a9b8f
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:38:08 2019 +1000

        Adding en_GB-large dictionary

    commit c94c6c39503750c820cdab533b4372695b61bad3
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:37:37 2019 +1000

        Fix en_US-large and en_CA-large hashes

    commit 84a847b39e7ad967d26434b0f6935211bb42010a
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:15:54 2019 +1000

        Adding en_AU-large dictionary

    commit b7d6e1ceebae98a5d7f3c6e119cb3816b879ed54
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:15:36 2019 +1000

        Adding en_CA-large dictionary

    commit c1857a6e054b60bbe51eda4a8c3c55285cf6092e
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:15:10 2019 +1000

        Adding en_US-large dictionary

    commit 898caa0eb9452992d5dc59cd82d267fcc8ad1133
    Author: Jai Flack <jaiflack@protonmail.com>
    Date:   Sun Aug 25 15:14:25 2019 +1000

        Modifying mkDictFromWordlist to allow for different src and destination
        file names

Provessor commented:
How's this @mmahut?
I can't find how to squash the first commit with the rest (I've tried twice as you can see in the commit history). If you could point me in the right direction with anything else that needs to be done, I'll get on it.
